packages/rabbitmq/src/rabbitmq.module.ts

```diff
@@ -78,6 +76,16 @@ export class RabbitMQModule
   }
 
   static async AmqpConnectionFactory(config: RabbitMQConfig) {
+    if (
+      typeof RabbitMQModule.connectionManager.getConnection(
```

Comment:
Do not check undefined like this, you can just assert that it is truth. Otherwise, if you feel that you want to check for null/undefined you can use lodash helpers